packages/one-app-bundler/package.json

```diff
@@ -55,7 +55,7 @@
     "file-loader": "^6.0.0",
     "folder-hash": "^3.3.0",
     "glob": "^7.1.3",
-    "holocron-module-register-webpack-plugin": "^1.2.0",
+    "holocron-module-register-webpack-plugin": "^1.2.2",
```

Comment:
Why didn't the yarn lockfile update? Was this done manually instead of via yarn?

Reply:
Updated. I ran `npm` instead of `yarn`.